### DISCUSSION TITLE
New Method vs Old Method

### DISCUSSION THREAD

**Discussion by jfischoff** [TheLastBen/fast-stable-diffusion#1016] [General]
I'm curious to know the pros and cons of the new method vs the old method. 

As I understand it, before the old way required generating a bunch of "class" images to use in the "prior preservation loss" metric when trying. 

The new method doesn't appear use "prior preservation loss". I'm curious to know why you made the change.

**Comment by TheLastBen** [comment]
The new method doesn't include an instance prompt composed of existing words in the dictionary, so it makes it harder to overfit the model which does the same job as class images which is limiting overfitting.

Now to further increase diversity of the output, I added another feature which is concept images, they do the same job as class images but more effectively as they have heavier weight than class images which is more suitable for stable diffusion.

Concept images will help greatly when training a style or a concept, but if you train on a specific face, don't use them.
